flang/CMakeLists.txt

```diff
@@ -248,7 +237,25 @@ else()
   include_directories(SYSTEM ${MLIR_TABLEGEN_OUTPUT_DIR})
 endif()
 
-option(FLANG_INCLUDE_RUNTIME "Build the runtime in-tree (deprecated; to be replaced with LLVM_ENABLE_RUNTIMES=flang-rt)" ON)
+set(FLANG_INCLUDE_RUNTIME_default ON)
+if ("flang-rt" IN_LIST LLVM_ENABLE_RUNTIMES)
+  set(FLANG_INCLUDE_RUNTIME_default OFF)
+endif ()
+option(FLANG_INCLUDE_RUNTIME "Build the runtime in-tree (deprecated; to be replaced with LLVM_ENABLE_RUNTIMES=flang-rt)" FLANG_INCLUDE_RUNTIME_default)
```

Comment:
`FLANG_INCLUDE_RUNTIME_default` -> `${FLANG_INCLUDE_RUNTIME_default}`.

Otherwise, this always evaluates to OFF for me, and I do not get the runtime to be built by default.

Reply:
Thanks, I think I didn't notice because I am always passing `-DLLVM_ENABLE_RUNTIMES=...`.